strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -80,6 +80,11 @@ def prepare_squeezed_state(self, r, phi, mode):
         self.circuit.loss(0.0, mode)
         self.circuit.squeeze(r, phi, mode)
 
+    def prepare_displaced_squeezed_state(self, alpha, r, phi, mode):
+        self.circuit.loss(0.0, mode)
```

Comment:
why is `loss` included here?

Reply:
Yep, that is how @nquesada does resets in the Gaussian backend